docs/ARCHITECTURE.rst

```diff
@@ -937,3 +937,48 @@ Algorithms
 The algorithms for each listing type can be found in the open-source
 `scality/Arsenal <https://github.com/scality/Arsenal>`__ repository, in
 `lib/algos/list <https://github.com/scality/Arsenal/tree/master/lib/algos/list>`__.
+
+Encryption
+===========
+
+With CloudServer, there are two possible methods of at-rest encryption.
+(1) We offer bucket level encryption where Scality CloudServer itself handles at-rest
+encryption for any object that is in an 'encrypted' bucket regardless of what
+the location-constraint for the data is and
+(2) If the location-constraint specified for the data is of type AWS,
+you can select to use AWS server side encryption.
+
+Example:
+--------
+
```

Comment:
Note that bucket level encryption is not available on the standard AWS S3 protocol. So normal AWS S3 clients will not provide the option to send a header when creating a bucket. We have created a simple tool to enable you to easily create an encrypted bucket.